src/static/riot/queues/list.tag

```diff
@@ -0,0 +1,282 @@
+                <em>No Queues Yet!</em>
+            </td>
+        </tr>
+        </tbody>
+        <tfoot>
+        <!-------------------------------------
+                  Pagination
+        ------------------------------------->
+        <tr if="{queues.length > 0 && _.get(pagination, 'next')}">
+            <th colspan="6">
+                <div class="ui right floated pagination menu" if="{queues.length > 0}">
+                    <a show="{!!_.get(pagination, 'previous')}" class="icon item" onclick="{previous_page}">
+                        <i class="left chevron icon"></i>
+                    </a>
+                    <div class="item">
+                        <label>{page}</label>
+                    </div>
+                    <a show="{!!_.get(pagination, 'next')}" class="icon item" onclick="{next_page}">
+                        <i class="right chevron icon"></i>
+                    </a>
+                </div>
+            </th>
+        </tr>
+        </tfoot>
+    </table>
+
+    <div class="ui modal" ref="modal">
+        <div class="header">
+            Queue Form
+        </div>
+        <div class="content">
+            <form class="ui form" ref="form">
+                <div class="ui active tab" data-tab="details">
+                    <div class="required field">
+                        <label>Name</label>
+                        <input name="name" placeholder="Name" ref="queue_name" value="{ _.get(selected_queue, 'name', '') }">
+                    </div>
+                    <div class="field">
+                        <div class="ui checkbox">
+                            <label>Is Public?</label>
+                            <input type="checkbox" ref="queue_public">
+                        </div>
+                    </div>
+                    <div class="field">
+                        <label>Collaborators</label>
+                        <div class="ui fluid search multiple selection dropdown collab-search">
+                            <input type="hidden" name="collaborators" ref="queue_collaborators">
+                            <i class="dropdown icon"></i>
+                            <div class="default text">Select Collaborator</div>
+                            <div class="menu">
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="actions">
+            <div class="ui primary button" onclick="{ handle_queue }">Submit</div>
+            <div class="ui basic red cancel button" onclick="{ close_modal }">Cancel</div>
+        </div>
+    </div>
+
+    <script>
+        var self = this
+        self.queues = []
+        self.selected_queue = {}
+        self.page = 1
+
+        self.one("mount", function () {
+            self.update_queues()
+            $(".ui.checkbox", self.root).checkbox()
+            $('.collab-search')
```

Comment:
`ref` instead

Reply:
Replaced JQuery references to use `self.refs`